specification/dimensions/chargesubcategory.md

```diff
@@ -0,0 +1,63 @@
+| Value format    | list-of-values |
+
+Allowed values for usage:
+
+| Value             | Description                                                                                                                                                                                                                                                                                                                                                |
+| :---------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Commitment Used   | Usage charges that are covered by pre-existing commitments, such as capacity commitments or commitment-based discounts.                                                                                                                                                              |
+| Commitment Not Used | Amortized usage charges for the portion of a commitment that has not been used. For example, if an organization has a commitment-based discount that is not fully utilized, the unused portion falls under this category. It highlights an area where the organization is not fully leveraging its commitments, which could be a lost cost-saving opportunity. |
+
+Allowed values for adjustments:
+
+| Value              | Description                                                                                                                                                                                                              |
+| :----------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Refund             | Negative charges that were previously billed and are being returned by the provider. Providers can have multiple types of refunds such as resolving a tax error or for returned or exchanged commitment-based discounts. |
+| Credit             | Negative charges granted by the provider for various scenarios, like negotiated benefits, usage discounts, or promotional credits.                                                                                       |
+| Rounding Error     | Positive or negative charges that are needed to ensure raw cost and usage data aggregations match the total cost on the invoice, which may be rounded.                                                                   |
+| General Adjustment | Positive or negative charges the provider applies that do not fall into other adjustment category values.                                                                                                                |
```

Comment:
What's an example of when someone would use General Adjustment?

Reply:
If a CSP under-charges over a period of time, and posts a positive charge for an adjusting amount in the subsequent period, I believe it would fall under `General Adjustment`. Not sure what negative charges could be created that are not `Refund` or `Credit`

Ex: Correcting previous bills where a SKU was given a lower price than intended, or applying a breach of private pricing spend commitment (true-up)

```suggestion
| General Adjustment | Positive or negative charges the provider applies to support contractual terms or to correct billing errors that do not fall into other adjustment category values                                                                                                               |
```

Reply:
"charges the provider applies that do not fall into other adjustment category values"

Reply:
@aqu-erp, does that also cover anything we haven't accounted for? That was the original intent.

Reply:
😆 I think `contractual terms` and `billing errors` also add color in a generic way that doesn't exclude anything, given everything on the invoice should be related to billing mechanics or described in a service agreement. I added the examples since the other descriptions had them as well, potentially seems less jarring this way.

Reply:
keeping the original and can be addressed post 1.0